### PR TITLE
feat: backflow

### PR DESCRIPTION
**Pull request by venjiang**
This change is [Reviewable](https://reviewable.io/reviews/yomorun/yomo/323)

### CODE REVIEW

File: core/connector.go [hunk 1]
```diff
@@ -48,24 +49,30 @@ type Connector interface {
 	AppName(connID string) (string, bool)
 	// LinkApp links the app and connection.
 	LinkApp(connID string, name string, observed []byte)
```
Comment (fanweixiao) [inline core/connector.go:51]: share we use `sfn` instead of `App` after we introduced `Source` here ?

**Review by fanweixiao — APPROVED** [review]
LGTM

File: core/connector.go (hunk 1 above)
Comment (venjiang) [reply core/connector.go:51]: Yeah, It's right.

File: source.go [hunk 2]
```diff
@@ -27,6 +27,8 @@ type Source interface {
 	SetErrorHandler(fn func(err error))
 	// WriteFrame writes a frame to the connection
 	WriteFrame(frm frame.Frame) error
+	// SetReceiveHandler set the observe handler function
```
Comment (wujunzhuo) [inline source.go:30]: suggest: `Experimental` & `Unstable`

**Review by wujunzhuo — APPROVED** [review]

**Review by xiaojian-hong — APPROVED** [review]

File: source.go (hunk 2 above)
Comment (venjiang) [reply source.go:30]: 😀

**Pull request closed without merging**